chart/compass/charts/kyma-environment-broker/templates/deployment.yaml

```diff
@@ -180,6 +180,26 @@ spec:
               value: "{{ .Values.provisioner.dumpRequests }}"
             - name: APP_DEFAULT_REQUEST_REGION
               value: "{{ .Values.broker.defaultRequestRegion }}"
+            - name: APP_AUDITLOG_URL
+              valueFrom:
+                configMapKeyRef:
+                  name: {{ .Values.global.auditlog.configMapName }}
```

Comment:
Why it is a global value, any need for that?

Reply:
I am reusing the existing value here. It has been used for gateway deployment too.